01-installing-and-running-ui/README.md

```diff
@@ -153,185 +183,396 @@ In the console you will see this message `Digitransit-ui available on port 8080`
 * `config.rt.js`
     * Replace the content of `config.rt.js` with the following code:
     ```
-    /* eslint-disable */
-    import configMerger from '../util/configMerger';
-
-    const CONFIG = 'rt';
-    const APP_TITLE = 'Reutlingen';
-    const APP_DESCRIPTION = 'stadtnavi - reutlingen';
-    const API_URL = process.env.API_URL || 'https://api.stadtnavi.de';
-    const MAP_URL = process.env.MAP_URL || 'https://tiles.stadtnavi.eu/streets/{z}/{x}/{y}{r}.png';
-    const SEMI_TRANSPARENT_MAP_URL = process.env.SEMI_TRANSPARENT_MAP_URL ||         'https://api.maptiler.com/maps/ffa4d49e-c68c-46c8-ab3f-60543337cecb/256/{z}/{x}/{y}.png?key=eA0drARBA1uPzLR6StGD';
-    const GEOCODING_BASE_URL = process.env.GEOCODING_BASE_URL || "https://photon.stadtnavi.eu/pelias/v1";
-    const LOCATIONIQ_API_KEY = process.env.LOCATIONIQ_API_KEY;
-    const YEAR = 1900 + new Date().getYear();
-    const STATIC_MESSAGE_URL =
+  /* eslint-disable */
+import configMerger from '../util/configMerger';
+
+const CONFIG = 'rt';
+const APP_TITLE = 'stadtnavi Reutlingen';
+const APP_DESCRIPTION = 'Gemeinsam Mobilität neu denken - die intermodale Verbindungssuche mit offenen, lokalen Daten';
+const API_URL = process.env.API_URL || 'https://api.stadtnavi.de';
+const MAP_URL = process.env.MAP_URL || 'https://tiles.stadtnavi.eu/streets/{z}/{x}/{y}{r}.png';
+const SEMI_TRANSPARENT_MAP_URL = process.env.SEMITRANSPARENT_MAP_URL || "https://tiles.stadtnavi.eu/satellite-overlay/{z}/{x}/{y}{r}.png";
+const GEOCODING_BASE_URL = process.env.GEOCODING_BASE_URL || "https://photon.stadtnavi.eu/pelias/v1";
+const YEAR = 1900 + new Date().getYear();
+const STATIC_MESSAGE_URL =
     process.env.STATIC_MESSAGE_URL ||
     '/assets/messages/message.hb.json';
 
-    const walttiConfig = require('./config.waltti.js').default;
+const walttiConfig = require('./config.waltti.js').default;
+
+const realtimeHbg = require('./realtimeUtils').default.hbg;
+const hostname = new URL(API_URL);
+realtimeHbg.mqtt = `wss://${hostname.host}/mqtt/`;
```

Comment:
Step 1 of the tutorial should explain the bare minimum. No realtime service, fare service, roadworks, echarging, weather station... Only features which will be available out-of-the-box using the otp-setup of step 2.